saffron/src/blob.rs

```diff
@@ -92,86 +92,13 @@ impl<G: CommitmentCurve> FieldBlob<G> {
     }
 }
 
-#[cfg(test)]
-mod blob_test_utils {
```

Comment:
this got copy/pasted over to `utils.rs` since it is useful for generating data in pretty much all cases